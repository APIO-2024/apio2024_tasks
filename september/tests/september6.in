txy4h26c1rm1uv8tr3eonahd67u8h56x
4
7 1
0 1 2 2 2 5
6 4 5 3 1 2
10 1
0 0 1 0 1 2 4 4 4
7 9 6 5 3 2 4 1 8
7 1
0 1 1 0 2 5
6 4 2 5 3 1
6 1
0 0 2 2 1
3 5 4 2 1
