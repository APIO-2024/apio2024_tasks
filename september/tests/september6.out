7ckgnn4wyi495puj3ibqf81dqvapyv6b
5
7
5
5
